eth/tracers/tracer.go

```diff
@@ -390,7 +390,7 @@ func New(code string) (*Tracer, error) {
 		return 1
 	})
 	tracer.vm.PushGlobalGoFunction("isPrecompiled", func(ctx *duktape.Context) int {
-		_, ok := vm.PrecompiledContractsByzantium[common.BytesToAddress(popSlice(ctx))]
+		_, ok := vm.PrecompiledContractsIstanbul[common.BytesToAddress(popSlice(ctx))]
```

Comment:
@karalabe I decided to switch it since we are adding a new precompile and we are almost-Istanbul but I am not sure about this change. Can you please confirm?

Reply:
I think that is an ok change. This adds a helper-method to the js-environment, and it is obviously already not quite correct (if executed pre-byzantium). This change does not really make it a lot more incorrect. 
A proper fix would be to look at the block number and chain config, but those are not accessible at this point, so would need a refactor which it out of scope for this PR, imo.